modules/executive/components/ProgressBar.tsx

```diff
@@ -30,7 +30,7 @@ function CircleNumber({ selected = false, num }: { selected?: boolean; num: numb
 }
 
 export default function ProgressBar({ step }: { step: number }): JSX.Element {
-  const stepTexts = ['Withdraw', 'Deposit', 'Vote'];
```

Comment:
Maybe we can just remove this Progress bar entirely? I think it's a bit redundant with the vote address(0) flow on the account page, and looks kinda bad because it will flash before it realizes Ive already voted.

Reply:
Small PR for this: <https://github.com/makerdao/governance-portal-v2/pull/1080>